applications/MeshingApplication/tests/Kratos_Execute_Meshing_Test.py

```diff
@@ -56,6 +53,12 @@ def __init__(self, ProjectParameters):
         # If we integrate it in the model part we cannot use combined solvers
         self.solver.AddDofs()
 
+        if self.problem_type  == "fluid":
+            self.Model = {self.ProjectParameters["problem_data"]["model_part_name"].GetString(): self.main_model_part} 
```

Comment:
why don't you use `Model` here too?

Reply:
it is supported in FluidDynamics?

Reply:
[seems like it](https://github.com/KratosMultiphysics/GiDInterface/blob/master/kratos.gid/apps/Fluid/python/KratosFluid.py#L58)

Reply:
I am not sure though why they still add the submodelparts manually
This should not be necessary right?

Reply:
@KratosMultiphysics/fluid-dynamics how should it be done?

Reply:
Ok

Reply:
Can you change it? , I am in the middle of a branch that takes too long to compile

Reply:
I can do it but I don't have mmg so probably I cannot run the test

Reply:
There is not any test of fluid currently, I move them to the Examples repository

Reply:
~~~sh
===== SERIAL EXECUTION =====
 |  /           |             
 ' /   __| _` | __|  _ \   __|
 . \  |   (   | |   (   |\__ \ 
_|\_\_|  \__,_|\__|\___/ ____/
           Multi-Physics 5.3.0-6aacbfd
Importing    KratosMeshingApplication
Initializing Kratos MeshingApplication... 
Importing    KratosExternalSolversApplication
Initializing KratosExternalSolversApplication... 
Importing    KratosFluidDynamicsApplication
Initializing KratosFluidDynamicsApplication... 
Importing    KratosStructuralMechanicsApplication
     KRATOS   ___|  |                   |                   |                     
            \___ \  __|  __| |   |  __| __| |   |  __| _` | |                   
                  | |   |    |   | (    |   |   | |   (   | |                     
            _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS     
TetrahedraReconnectUtility process is not compiled and the corresponding tests will not be executed
MMG process is not compiled and the corresponding tests will not be executed
TetrahedraReconnectUtility process is not compiled and the corresponding tests will not be executed
MMG process is not compiled and the corresponding tests will not be executed
[Warning]: "all" test suite is empty

~~~

should I still commit the change?

Reply:
I think me can leave this to other occasion, without fluid cases to test I think is meaningless to perform any other change until there is a case like that

Reply:
ok